### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by ngeelani48**
**In this branch following functionalities are added:**

- Added any missing JSX elements to the project.
- Used the CSS rules detailed in Zeplin to style the website.
- Match the final design to Zeplin's design.

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @ngeelani48 🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that has a README that is not professional. Here are key points to consider when documenting your README.

- Kindly delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. ✖️ 

image

- Kindly remove the example text on your Future Features section

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Book.js [hunk 1]
```diff
@@ -1,30 +1,91 @@
...
+          </p>
+          <button
+            className="update-progress"
+            type="button"
+            title="UPDATE PROGRESS"
+          >
+            UPDATE PROGRESS
+          </button>
+        </div>
+      </div>
```
Comment (julie-ify) [inline src/components/Book.js:80]: - [ ] Kindly improve the styling of the book card to match the design. As noted in the requirement we are expected to make an EXACT copy of the Zeplin design.

  - Kindly use the correct fonts. Some of your fonts do not match the ones in the design
  - Kindly note the spacing between the elements as well, for example, the right section of the card containing the "Update Progress" button should be moved more toward the left to give some space from the right border.
  - Kindly also add some side padding the button.

  Zeplin design:
  iosap

  <hr/>

  Your design:
  sjaia

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @ngeelani48,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @ngeelani48 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## HIghlights
- No linter errors :heavy_check_mark: 
- Well-detailed README :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**